subsys/usb/usb_device.c

```diff
@@ -143,6 +143,15 @@ static struct usb_dev_priv {
 	bool remote_wakeup;
 } usb_dev;
 
+/** Setup packet definition used to read raw data from USB line */
+struct usb_setup_packet_packed {
```

Comment:
Shouldn't this be in a header file?

Reply:
This is just for one purpose I don't think we should make it public. ```struct usb_setup_packet``` is enough for the user.

Reply:
OK, agreed.